parts/linux/cloud-init/artifacts/reconcile-private-hosts.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
what is the difference to #!/bin/bash that we're using in other places?

Reply:
it has the same effect in this situation. And I notice we're using these two style interchangeably within artifacts.

Reply:
good to know. but /bin/bash is used more.

Reply:
OK. Changed to /bin/bash